scripts/deploy-validator.sh

```diff
@@ -114,6 +110,7 @@ PUBLIC_KEY=$(linera-server generate --validators validator-config.toml)
 echo "Validator setup completed successfully."
 echo "Starting docker compose..."
 
+export LINERA_IMAGE="us-docker.pkg.dev/linera-io-dev/linera-public-registry/linera:$BRANCH_NAME"
```

Comment:
For actual validators, should we do `release_$BRANCH_NAME`? And make sure that we tag the image with the version of the branch we want validators to use with this release tag?

Reply:
We don't currently have the machinery to do `$BRANCH_NAME_release`  in the workflow. We can add it when we do. 

I think we can keep it simple for now.

Reply:
Think about this as well. With the _current_ status quo validators are checking out the latest version of the branch anyway, there is no release tag which they are using.

Reply:
Sounds good! We can add it back once we need it